.github/workflows/release.yaml

```diff
@@ -0,0 +1,46 @@
+name: release
+on:
+  push:
+    tags:
+      - "v*"
+jobs:
+  goreleaser:
+    name: Release a new version
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "1.16.x"
+
+      - uses: docker/setup-qemu-action@v1
+      - uses: docker/setup-buildx-action@v1
+      - uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
```

Comment:
I've created the `GoReleaser` environment, so you'll need to [reference](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-a-single-environment-name) that in this workflow.

I also added environment level secrets for `DOCKERHUB_TOKEN` and `PACKAGECLOUD_TOKEN`. Lemme know if you have any issues with these secrets.
Let's hardcode the docker username to `golangmigrate`. I don't see any point in encrypting that.

Reply:
Done! Agree at the no point of the encryption of the username. It's all in Docker Hub, anyone can see it..